docs/execute-methods.md

```diff
@@ -1228,3 +1228,19 @@ Xcode must be at version 14.3+ and iOS must be at version 16.4+.
 > **Warning**
 > Do not forget to reset the simulated geolocation value after your automated test is finished.
 > If the value is not reset explcitly then the simulated one will remain until the next device restart.
+
+### mobile: getAppStrings
+
+Retrieves string resources for the given app language. An error is thrown if strings cannot be fetched or no strings exist
+for the given language abbreviation
```

Comment:
`since XCUITest driver 4.22.0`?

Reply:
I think in this project it is not necessary since the documentation is generated for each version

Reply:
Ah, correct. Missed the difference when I switched projects..